src/org/ods/component/Context.groovy

```diff
@@ -340,10 +340,12 @@ class Context implements IContext {
     config.groupId
   }
 
+  @NonCPS
```

Comment:
What is this annotation for?

Reply:
To be able to call this method from a constructor (Jenkins CPS magic ...).